src/Image.php

```diff
@@ -609,7 +609,7 @@ protected function gc()
                 if (file_exists($file->getPathname()) && $file->getMTime() < $expire) {
                     // only deletes files ending with $this->suffix
                     if (substr($file->getFilename(), -($suffixLength)) == $this->suffix) {
-                        unlink($file->getPathname());
+                        @unlink($file->getPathname());
```

Comment:
Wrap this in an error handler, please; error suppression is expensive, and makes testing more difficult.